src/vs/workbench/contrib/notebook/browser/contrib/find/notebookFindReplaceWidget.ts

```diff
@@ -159,22 +160,18 @@ class NotebookFindInput extends FindInput {
 
 	constructor(
 		readonly filters: NotebookFindFilters,
-		contextKeyService: IContextKeyService,
 		readonly contextMenuService: IContextMenuService,
 		readonly instantiationService: IInstantiationService,
-		parent: HTMLElement | null,
-		contextViewProvider: IContextViewProvider,
-		options: IFindInputOptions
+		options: IFindInputOptions,
+		filterStartVisiblitity = true // TODO: create notebook find options to define this
```

Comment:
Do we then want to have this `filterStartVisiblitity` on `SearchFindInput` then?